lib/ui/mockData/MockUserProfileViewModel.dart

```diff
@@ -45,18 +48,18 @@ class MockUserProfileViewModel {
 }
 
 class MockUserProfileListItemViewModel {
-  static UserProfileListItemViewModel build(index) {
-    return UserProfileListItemViewModel(
-      title: _mockActionTitle[index],
+  static ProfileListItemViewModel build(index) {
+    return ProfileListItemViewModel(
+      title: Intl.message(_mockActionTitle[index]),
       icon: _mockActionIcon[index],
       onTap: () => _mockItemTap(),
       isDestructive: index != 7 ? false : true,
     );
   }
 
-  static UserProfileListItemViewModel buildLarger(index) {
-    return UserProfileListItemViewModel(
-      title: _mockActionTitleLarger[index],
+  static ProfileListItemViewModel buildLarger(index) {
+    return ProfileListItemViewModel(
+      title: Intl.message(_mockActionTitleLarger[index]),
```

Comment:
That should be removed, it's in mock area, probably you got it from rebase.

Reply:
ah nuts
will remove

Reply:
done